### DISCUSSION TITLE
Editare i file YAML in Core

### DISCUSSION THREAD

**Discussion by riccionese** [indomus/forum#830] [GENERALE - Discussioni generiche]
Ciao a tutti, 
io ho HA Core installato su pi4.
finora ho sempre gestito li file YAML con il mio buon vecchio "vi", collegandomi in `ssh` direttamente con il Pi4.
E' una cosa che faccio da sempre ma immagino ci siano modi più furbi/eleganti/pratici, soprattutto per gestire le versioni dei file che create modificate (git, immagino)
Come primo step avevo condiviso in rete la directory `.homeassistant` per raggiungere i file senza dovermi collegare ma non è un gran vantaggio.
Per cui vorrei sapere voi come fate e che suggerimenti avete.

**Comment by snakuzzo** [comment]
Io edito quasi sempre da remoto con VSCode sul PC che mi pare + pugin [RemoteSSH](https://code.visualstudio.com/docs/remote/ssh) ed [Home Assistant Config Helper](https://marketplace.visualstudio.com/items?itemName=keesschollaart.vscode-home-assistant).

**Reply by riccionese** [reply]
adesso cerco tutti i nomi che hai scritto e provo a tradurre :-) 
ciaux !

**Reply by snakuzzo** [reply]
dai un'occhiata anche al [NOSTRO ARTICOLO](https://indomus.it/progetti/modificare-i-file-di-configurazione-home-assistant-core-su-raspbian-con-visual-studio-code-vscode/)
